packages/vscode/package.json

```
@@ -145,6 +145,12 @@
             "/path/to/your/schema.prisma"
           ],
           "description": "If you have a Prisma schema file in a custom path, you will need to provide said path `/path/to/your/schema.prisma` to run generate"
+        },
+        "prisma.pdpNotification": {
+          "title": "Pdp Notification",
+          "type": "boolean",
+          "default": "true",
+          "description": "..."
```

```suggestion
        "prisma.pdpNotification": {
          "title": "Display promotion notification for Prisma Data Platform.",
          "type": "boolean",
          "default": "true",
          "description": "If checked, a notification will be displayed, once, on the next activation of the Prisma extension only."
```
@janpio title + description